IETF.js

```diff
@@ -81,11 +81,15 @@ function doWeb(doc, url) {
 		}
 		newItem.reportNumber = "RFC " + ZU.xpathText(metadataDoc, "//meta[@name='DC.Identifier']/@content").split(":")[3];
 		newItem.institution = "IETF";
-		var abstractContent = ZU.xpathText(metadataDoc, "//meta[@name='DC.Description.Abstract']/@content").split("[");
-		newItem.abstractNote = abstractContent[0].replace(/\n/g, " ");
-		newItem.reportType = abstractContent[1].split("]")[0];
+		let abstractContent = ZU.xpathText(metadataDoc, "//meta[@name='DC.Description.Abstract']/@content");
+		newItem.abstractNote = abstractContent.replace(/\n/g, " ");
+		
+		let regexp_type = /\[([^\]]+)\]$/;
```

Comment:
`let regexp_type` → `let re`, but you want to use a regexp with groups for both the abstract itself and the type at the end.

Reply:
If I understood this, I think that I have addressed the variable change correctly.